Shared/Samples/Download vector tiles to local cache/DownloadVectorTilesToLocalCacheView.swift

```diff
@@ -96,11 +116,20 @@ struct DownloadVectorTilesToLocalCacheView: View {
                                 guard let extent = mapView.envelope(fromViewRect: viewRect) else { return }
                                 
                                 // Downloads the vector tiles.
-                                await model.downloadVectorTiles(extent: extent)
-                                // Sets downloading to false when the download finishes.
-                                isDownloading = false
+                                do {
```

Comment:
I move the error handling into the view's task modifier so that the model doesn't need to know about a view. See discussion here: https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/42#discussion_r942747195